application/src/js/cms/_scroll_past_sticky_status_banner.js

```diff
@@ -0,0 +1,28 @@
+;(
+    function () {
+        var errorBox = document.querySelector('.flash-message .alert-box.error');
+
+        if (!isUndefinedOrNull(errorBox)) {
+            var errors = errorBox.querySelectorAll('a')
+
+            var scrollForStickyHeader = function (event) {
+                window.setTimeout(function () {
```

Comment:
Can we add a comment to say why the timeout function is used (ie to allow the default event to fire first)?

Reply:
Done